File: paciente-backend/Models/Paciente.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApplication1.Models
{
    [Table("paciente")]
    public class Paciente
    {
        [Key]
        public long Id { get; set; }
        public required string NomeCompleto { get; set; }
        public DateTime DataNascimento { get; set; }
        public Char Sexo {  get; set; }
        public int Idade { get; set; }
    }
}

File: paciente-frontend/src/app/pages/pacientes/model/paciente.ts
import { Anamnese } from "./anamnese";

export interface Paciente {
  id?: number;
  nomeCompleto: string;
  dataNascimento: any;
  sexo: string;
  idade: number;
  anamnese: Anamnese;
}

export interface PacienteListagem {
  items: Paciente[];
  count: number;
}